src/autopas/options/DataLayoutOption.h

```diff
@@ -11,7 +11,7 @@
 #include "autopas/options/Option.h"
 
 namespace autopas {
-
+inline namespace options {
```

Comment:
We need [nested inline namespaces](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p1094r2.html) so lets upgrade to C++20 😈  /s

Reply:
yes, to skrew up doxygen even more xD